ctags.yaml

```diff
@@ -0,0 +1,6 @@
+name: ctags
+sources:
+  - type: git
+    url: https://github.com/universal-ctags/ctags
+    # No releases! Nice.
+    commit: 42b74d5dbe70dc315e650fb4d2b5acaa19db114e
```

Comment:
Not a big fan of adding this dependency without something to auto-update it. Of course this repo already has a Git-based auto-update script but it would be nice to migrate that into the other auto-update script I maintain, namely https://github.com/endlessm/flatpak-external-data-checker/ …